driver/run_components.py

```diff
@@ -41,18 +46,33 @@ def get_executable(build: str, rel_path: Path):
         #   '<repo-root>/builds/<buildname>/bin'.
         build_dir = util.BUILDS_DIR / build / "bin"
         if not build_dir.exists():
-            returncodes.exit_with_driver_input_error(
-                f"Could not find build '{build}' at {build_dir}. "
-                f"Please run './build.py {build}'.")
+            raise IncompleteBuildError(
+                f"Could not find build '{build}' at {build_dir}.")
```

Comment:
It seems the error message of `IncompleteBuildError` is never used? Then I would instantiate it without arguments, or more minimalistically with `IncompleteBuildError(build)`. (For example, this is how KeyError works in Python). Also for other place(s) where this exception is raised.

Reply:
This one?
```Python
except IncompleteBuildError as err:
```

What is the problem there?

Reply:
We discussed this further in chat and decided to drop support for external build that cause problems here. I think this resolves this conversation (unresolve it if you disagree).